cli/app/app.go

```diff
@@ -1260,6 +1264,8 @@ func Up(c *cli.Context) {
 		dcCreated, err := client.Deployments(api.NamespaceDefault).Create(dc)
 		if err != nil {
 			logrus.Fatalf("Failed to create deployment controller %s: ", k, err)
+		} else {
+			fmt.Println("Deployment controller " + k + " has been created.")
```

Comment:
We need the same validation in convert for checking -f and --bundle aren't specified at the same time. Make that a function and call it from both places.